.github/workflows/PRLabeling.yml

```
@@ -0,0 +1,29 @@
+
+      - name: Fetch PR details
+        id: pr_details
+        run: |
+          CHANGED_LINES=$(jq '. | .additions + .deletions' <<< "${{ toJson(github.event.pull_request) }}")
+          echo "::set-output name=changed_lines::$CHANGED_LINES"
+
+      - name: Label PR
+        uses: actions-ecosystem/action-add-labels@v1
+        with:
+          labels: |
+            ${{ steps.pr_details.outputs.changed_lines }} < 50 && echo "small" || 
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)